plugin-gradle/CHANGES.md

```diff
@@ -2,6 +2,8 @@
 
 ### Version 3.5.0-SNAPSHOT - TBD ([javadoc](https://diffplug.github.io/spotless/javadoc/snapshot/), [snapshot](https://oss.sonatype.org/content/repositories/snapshots/com/diffplug/spotless/spotless-plugin-gradle/))
 
+* Changed importOrder interface from array to varargs ([#125](https://github.com/diffplug/spotless/issues/125))
```

Comment:
I think it would look a little nicer if "importOrder" was wrapped in backticks (\`), like "\`importOrder\`", which would make it render as `importOrder`, better highlighting that it's code-related.